rotkehlchen/exchanges/bitstamp.py

```diff
@@ -188,12 +189,12 @@ def query_balances(self) -> Tuple[Optional[Dict[Asset, Dict[str, FVal]]], str]:
                 )
                 continue
 
-            asset_balance[asset] = {
-                'amount': amount,
-                'usd_value': amount * usd_price,
-            }
+            asset_balance[asset] = Balance(
+                amount=amount,
+                usd_value=amount * usd_price,
+            )
 
-        return asset_balance, ''
+        return dict(asset_balance), ''
```

Comment:
why do you need to cast to dict here. In this exchange I don't see you using a `defaultdict`